src/ucp/wireup/select.c

```diff
@@ -490,9 +492,12 @@ ucp_wireup_add_lane_desc(const ucp_wireup_select_info_t *select_info,
             (lane_desc->path_index == select_info->path_index))
         {
             lane = lane_desc - select_ctx->lane_descs;
-            ucs_assertv_always(dst_md_index == lane_desc->dst_md_index,
+            ucs_assertv_always(dst_dev_index == lane_desc->dst_dev_index,
                                "lane[%d].dst_md_index=%d, dst_md_index=%d",
                                lane, lane_desc->dst_md_index, dst_md_index);
+            ucs_assertv_always(dst_md_index == lane_desc->dst_md_index,
+                               "lane[%d].dst_dev_index=%d, dst_dev_index=%d",
```

Comment:
traces are incorrect, you mention dst_dev for dst_md and vice versa

Reply:
fixed